modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/blueprint/test/SXPBlueprintSearchResultTest.java

```diff
@@ -2021,54 +2021,34 @@ public void testTextMatchOverMultipleFields_crossFields() throws Exception {
 	@Test
 	public void testTextMatchOverMultipleFields_mostFields() throws Exception {
 		_journalArticleBuilder.setTitle(
```

Comment:
Insted of add a new assertSearch you should add

```
_updateConfigurationJSON(
			"queryConfiguration", JSONUtil.put("applyIndexerClauses", false));
```
It will disable the default clauses from elasticsearch and will be possible to prove that this element really works

Reply:
Done!